ci-operator/templates/openshift/endurance/endurance-install.yaml

```diff
@@ -377,8 +377,8 @@ objects:
       - /bin/sh
       - -c
       - |
-        #!/bin/sh -x
-
+        #!/bin/sh
+        set -euo pipefail
```

Comment:
do we need to move to Bash shebangs for `pipefail`?  I don't know if it's POSIX

Reply:
we should make the header of the files consistent

Reply:
https://github.com/openshift/release/pull/6718/files didn't

Reply:
(didn't move to /bin/bash i mean)

Reply:
No pipefail [in POSIX][1].  I think we should move to `#!/bin/bash`.

[1]: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_25